cmd/minikube/cmd/start.go

```diff
@@ -75,8 +75,10 @@ var (
 var startCmd = &cobra.Command{
 	Use:   commandName,
 	Short: "Starts a local OpenShift cluster.",
-	Long: `Starts a local OpenShift cluster using Virtualbox. This command
-assumes you already have Virtualbox installed.`,
+	Long: `Starts a local OpenShift cluster using VirtualBox. You must have VirtualBox installed
```

Comment:
"Starts a single node OpenShift cluster running on the selected hypervisor"

VirtualBox is only needed if it is the selected hypervisor.